setup/setup.sh

```diff
@@ -1,2 +1,10 @@
 #!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 pip3 install requests furl bs4 lxml
```

Comment:
I recommend replacing this with `pip3 install -r requirements.txt` which will then mean that dependency changes will only need to reflected in one file, `requirements.txt`. This file is fairly standard for projects https://medium.com/@boscacci/why-and-how-to-make-a-requirements-txt-f329c685181e

Reply:
Just switched both scripts to that.